homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -13,7 +13,8 @@
     ATTR_CODE, ATTR_CODE_FORMAT, ATTR_ENTITY_ID, SERVICE_ALARM_TRIGGER,
     SERVICE_ALARM_DISARM, SERVICE_ALARM_ARM_HOME, SERVICE_ALARM_ARM_AWAY,
     SERVICE_ALARM_ARM_NIGHT, SERVICE_ALARM_ARM_CUSTOM_BYPASS)
-from homeassistant.helpers.config_validation import PLATFORM_SCHEMA  # noqa
+from homeassistant.helpers.config_validation import (
+    COMPONENT_SCHEMA, PLATFORM_SCHEMA_2 as PLATFORM_SCHEMA)  # noqa
```

Comment:
Importing platform schema here is wrong ? It should have allow_extra set ?

Reply:
We now expose COMPONENT_SCHEMA with has allow_extra set, and PLATFORM_SCHEMA which does not. PLATFORM_SCHEMA may be used by platforms implementing the alarm component.